### PR TITLE
Add tests for useWindowSize

### PR DESCRIPTION
**Pull request by ankithkonda**
Referencing issue - #101  

My suggestion would be to create a folder for each hook and have any related file in there, including tests.  

In the spirit of using TS, I've used TS for the test as well. 

Not sure if this is the best practice for testing hooks. let me know what you guys think

### CODE REVIEW

**Comment by wardoost** [comment]
Looks like a good start 👍 We could we use [react-hooks-testing-library](https://github.com/mpeyper/react-hooks-testing-library) which will probably allow us to only have 1 test file per hook? I would also put all tests in a `__tests__` folder to align it with the `__stories__` folder.

**Comment by ankithkonda** [comment]
good call! I've updated to __tests__ and the test now uses [react-hooks-testing-library](https://github.com/mpeyper/react-hooks-testing-library) :)

File: jest.config.js [hunk 1]
```diff
@@ -0,0 +1,185 @@
+// For a detailed explanation regarding each configuration property, visit:
```
Comment (wardoost) [inline jest.config.js]: Do we need all these comments? It seems more clear to me to only include used config options

File: package.json [hunk 2]
```diff
@@ -57,15 +60,19 @@
...
     "fork-ts-checker-webpack-plugin": "1.0.1",
     "gh-pages": "2.0.1",
+    "jest": "^24.7.1",
     "keyboardjs": "2.5.1",
     "markdown-loader": "5.0.0",
     "react": "16.8.4",
     "react-dom": "16.8.4",
+    "react-hooks-testing-library": "^0.4.0",
     "react-spring": "6.1.10",
+    "react-testing-library": "^6.1.2",
```
Comment (wardoost) [inline package.json]: Not a direct dependency anymore?

File: src/__tests__/useWindowSize.test.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
```
Comment (wardoost) [inline src/__tests__/useWindowSize.test.tsx]: Don't have to import React

File: src/__tests__/useWindowSize.test.tsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+import useWindowSize from "../useWindowSize"
+import { renderHook, cleanup, act } from 'react-hooks-testing-library'
```
Comment (wardoost) [inline src/__tests__/useWindowSize.test.tsx]: `cleanup` not used

File: jest.config.js (hunk 1 above)
Comment (ankithkonda) [reply jest.config.js]: Agreed!

File: package.json (hunk 2 above)
Comment (ankithkonda) [reply package.json]: Ah forgot to remove this from the first commit

File: src/__tests__/useWindowSize.test.tsx (hunk 3 above)
Comment (ankithkonda) [reply src/__tests__/useWindowSize.test.tsx]: woops forgot this too!

File: src/__tests__/useWindowSize.test.tsx (hunk 4 above)
Comment (ankithkonda) [reply src/__tests__/useWindowSize.test.tsx]: Added cleanup according to the example on their readme

**Comment by ankithkonda** [comment]
All done :)

**Comment by wardoost** [comment]
Looks great, would you add a `test:watch` script to `package.json`?

**Comment by ankithkonda** [comment]
👍done! 👁

**Comment by wardoost** [comment]
🙏

**Pull request merged**